service/src/main/java/org/databiosphere/workspacedataservice/config/ClockConfig.java

```diff
@@ -0,0 +1,18 @@
+package org.databiosphere.workspacedataservice.config;
+
+import java.time.InstantSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ClockConfig {
+
+  /**
+   * Expose {@link InstantSource#system())} as a bean to provide an override seam for tests that
+   * need to verify clock behavior.
+   */
+  @Bean
+  InstantSource instanceSource() {
```

Comment:
Was this intended to be `instantSource`?

```suggestion
  InstantSource instantSource() {
```

Reply:
Yes, thanks for spotting!  Fixed in 87bdb44a5691af001b2470c0494d77b26a868848